File: templates/colors.py
colors_template="""
-- {{ colorscheme.name }} colorscheme
-- Repo: {{ colorscheme.repo }}
-- Maintainer: {{ colorscheme.author }} <"{{ colorscheme.contact }}">
-- License: {{ colorscheme.license }}

require('{{ colorscheme.name }}').load()
"""

File: nvim/template/colors.go
package template

func Colors() string {
	return `-- {{title (mustRegexFind "^[a-z]+" .Name)}} colorscheme
-- Repo: {{.Repo}}
-- Maintainer: {{title .Author}} <{{.Contact}}>
-- License: {{.License}}
require("{{mustRegexFind "^[a-z]+" .Name}}").load("{{.Name}}")`
}
